### DISCUSSION TITLE
CSV download of all source feeds on transit.land?

### DISCUSSION THREAD

**Discussion by westontrillium** [transitland/transitland#438] [Q&A] [answered]
Hello!

I am currently working on auditing presence of feeds Trillium hosts in public GTFS repositories and was wondering if transit.land had a downloadable CSV file or something similar for all feeds in the database?

Thanks!

**Comment by drewda** [comment, marked as answer]
No, Transitland does not automatically output CSVs listing feeds.

But we do have some debugging scripts that our data team use when reviewing many feed records. I'll be glad to follow up on that and see if one of those can be used to generate a CSV with all the Trillium feeds.
